### PR TITLE
Update AKO README for service type node port and Openshift

### PR DESCRIPTION
**Pull request by monotosh-avi**
- Initial readme changes for node port and openshift support
- Other docs like faq, values  would be added in next PRs

### CODE REVIEW

File: docs/AKO/README.md [hunk 1]
```diff
@@ -143,3 +143,42 @@ Please refer to this [page](objects.md) for details on how AKO interprets the ku
...
+Follow the steps 1 to 2, given in section [Pre-requisites](https://github.com/avinetworks/avi-helm-charts/tree/master/docs/AKO#pre-requisites). Additionaly following points have to be noted for openshift environment.
+1. Make Sure Openshift version is >= 4.4
+2. Openshift routes and services of type load balancer are suppported in AKO
+3. Ingresses, if created in the openshift cluster won't be handled by AKO.
+4. serviceType should be **NodePort** in values.yaml. Routes should use services of type NodePort as backends.
+
+Follow the same steps mentioned in [install-using-helm](https://github.com/avinetworks/avi-helm-charts/tree/master/docs/AKO#install-using-helm) to install AKO in an openshift cluster.
+
+#### Features of Openshift Route supported in AKO
+AKO suports the following features of openshift route. These features are currently in **beta** phase.
```
Comment (sudswasavi) [inline docs/AKO/README.md:175]: supports

File: docs/AKO/README.md [hunk 2]
```diff
@@ -143,3 +143,42 @@ Please refer to this [page](objects.md) for details on how AKO interprets the ku
...
+1. Insecure Routes.
+2. Insecure Routes with alternate backends.
+3. Secure routes with edge termination policy.
+
+#### Features of Openshift Route to be supported in upcoming releases AKO
```
Comment (sudswasavi) [inline docs/AKO/README.md:180]: releases of*

File: docs/AKO/README.md [hunk 3]
```diff
@@ -143,3 +143,42 @@ Please refer to this [page](objects.md) for details on how AKO interprets the ku
...
 For some frequently asked question refer [here](faq.md) 
+
+## *__Features in Beta Release__*
+> All the features mentioned in the subsequent sections are only part of the beta release and not present in the stable releases.
+
+### Node Port
+
+Service of type node port can be used to send traffic to the pods using nodeports. This can be used where the option of static IP in VRF Context is not feasible.
+
+A new parameter serviceType has been introduced as config option in AKO's values.yaml. To use this feature, set the value of the parameter to **NodePort**. Additionally disableStaticRouteSync should be set to "true".
```
[the remark this answers is not in the document]
Comment (parimanur) [reply docs/AKO/README.md:154]: Functionality-wise it currently ignores the value ( No-Op even if the value is set to True )

[the remark this answers is not in the document]
Comment (monotosh-avi) [reply docs/AKO/README.md]: Removed the line

File: docs/AKO/README.md [hunk 4]
```diff
@@ -143,3 +143,42 @@ Please refer to this [page](objects.md) for details on how AKO interprets the ku
...
+
+### Node Port
+
+Service of type node port can be used to send traffic to the pods using nodeports. This can be used where the option of static IP in VRF Context is not feasible.
+
+A new parameter serviceType has been introduced as config option in AKO's values.yaml. To use this feature, set the value of the parameter to **NodePort**.
+
+| **Parameter** | **Description** | **Default** |
+| --- | --- | --- |
+| `configs.serviceType` | Type of Service to be used as backend for Routes/Ingresses | ClusterIP |
```
Comment (apalsule) [inline docs/AKO/README.md:158]: serviceType also applies to service type LB.

**Comment by sudswasavi** [comment]
@apalsule its automatic.

File: docs/AKO/README.md (hunk 4 above)
Comment (monotosh-avi) [reply docs/AKO/README.md:158]: Added a description bellow for Service Type LB

**Pull request merged**